src/main/java/stream/flarebot/flarebot/commands/moderation/PurgeCommand.java

```diff
@@ -115,7 +117,7 @@ public String getDescription() {
 
     @Override
     public String getUsage() {
-        return "`{%}purge <messages>` - Purges a certain amount of messages";
+        return "`{%}purge <'all'/user> <amount>` - Purges a certain amount of messages.";
```

Comment:
we don't have a `.` on any other usages

Reply:
We do on like info
https://github.com/FlareBot/FlareBot/blob/master/src/main/java/stream/flarebot/flarebot/commands/general/InfoCommand.java#L81
We should have this as a standard.